docs/User-Research/2021, Summer - Communicating Errors in TDP & Validations With Regional Staff.md

```diff
@@ -0,0 +1,370 @@
+- [Specificity and putting the *right* data points in front of the right grantee is key to their ability to effectively act on an error](#specificity-and-putting-the-right-data-points-in-front-of-the-right-grantee-is-key-to-their-ability-to-effectively-act-on-an-error)
+
+---
+
+### Regional staff have the closest OFA-side view into the makeup of grantee teams, but that view is not always complete.
+
+Regional staff participants told us that while they're often at the forefront of contact between OFA and Grantees, they don't have comprehensive information into the full makeup of those grantee teams that they work with. Like OFA analysts, their contact tends to be with some subset of grantee staff. However, regional staff shared that they will usually know who the grantee program director is, as well as have frequent contact with whoever is taking the lead on data cleansing/validation. They compared the dynamic to a "social club", where transparency into membership is dependent on referrals from those who are already in it. 
+
+**Project Impact(s):**
+
+As there is no requirement for grantees to submit verifiable data about their teams, it seems likely that this "social club" dynamic will need to continue in regard to requests for TDP access. Ways we can investigate to add security into that include:
+
+- Giving grantee program directors a standardized way to submit a request for access for somebody on their team within TDP.
+- Requiring that non-director grantee staff have been authorized by their program director (via the above) to receive the access they're requesting.
+- Clearly displaying information about new users requesting access for the regional staff who will be approving them. (e.g email address, associated STT, associated/referring program director, etc...)
+
+---
+
+### We documented "error patterns"  that may be utilized throughout error guidance in TDP.
+
+**Naming Patterns**
+
+Plain Language Error Names
+
+- Affords glanceability (and potentially referenceability) to a user who might not be familiar with legacy error codes, or who might not have them committed to memory.
+
+Legacy Error Names
+
+- Largely for those either playing a role in designing their own grantee level systems or those already highly familiar with existing error codes. Draws a through line showing which legacy errors map to newer, plainer language TDP errors. Can be used alongside the newer error names.
+
+**Scope Patterns**
+
+Split Errors
+
+- There seems to be potential to make certain legacy TDRS errors more actionable by splitting them into several variations. One example being T1-515 (Update Indicator is not valid). In TDP we could provide several variations that better identify what's wrong with the update indicator and better contextualize what will happen to previously submitted data as a result of correcting it in different ways (e.g. D, U, N).
```

Comment:
I think we should use a different example here, since this error is related to partial submissions which we are moving away from.